compiler/src/dotty/tools/dotc/transform/CompleteJavaEnums.scala

```diff
@@ -117,12 +116,13 @@ class CompleteJavaEnums extends MiniPhase with InfoTransformer { thisPhase =>
     && cls.owner.owner.linkedClass.derivesFromJavaEnum
 
   @threadUnsafe
-  private lazy val enumCaseOrdinals: mutable.Map[Symbol, Int] = mutable.AnyRefMap.empty
+  private lazy val enumCaseOrdinals: MutableSymbolMap[Int] = newMutableSymbolMap
```

Comment:
None of the existing MutableSymbolMap are lazy so I think it's fine to do without the lazy here.

Reply:
ok will do